grpc-example/grpc-example-java/hello-proxy-impl/pom.xml

```diff
@@ -59,6 +59,26 @@
             <artifactId>akka-http2-support_${scala.binary.version}</artifactId>
             <version>${akka.http.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-http-core_${scala.binary.version}</artifactId>
+            <version>${akka.http.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-http_${scala.binary.version}</artifactId>
+            <version>${akka.http.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-parsing_${scala.binary.version}</artifactId>
+            <version>${akka.http.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-http-spray-json_${scala.binary.version}</artifactId>
+            <version>${akka.http.version}</version>
+        </dependency>
```

Comment:
This pom should have the play overrides too.

Reply:
what do you mean?

Reply:
( I think you added the missing play overrides in this pom on your last commit :-) )